### DISCUSSION TITLE
Docs: Add more cross linking with the Astro Docs

### DISCUSSION THREAD

**Discussion by mProjectsCode** [withastro/starlight#1072] [Feature Requests]
### What version of `starlight` are you using?

0.12.0

### What is your idea?

On Starlight docs pages, add more links to related Astro docs pages. 

### Why is this feature necessary?

Since Starlight is built on top of Astro, Starlight requires knowledge of Astro. Adding all the necessary knowledge to the Starlight docs would result in a lot of duplication and would make the docs harder to maintain.

### Do you have examples of this feature in other projects?

The Starlight docs page for [markdown](https://starlight.astro.build/guides/authoring-content/) only links to a specific section of the [Astro docs page for markdown](https://docs.astro.build/en/guides/markdown-content/) in one place under [Advanced Configuration](https://starlight.astro.build/guides/authoring-content/#advanced-markdown-and-mdx-configuration). I think that it would be an improvement to add a bit more crosslinking to the Starlight Docs.

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by mProjectsCode** [comment]
I am not a docs writer and relatively new to Starlight, but i can try to write a PR for this.

**Comment by delucis** [comment]
Thanks for the suggestion @mProjectsCode! I think this could definitely be helpful wherever there are places where the Astro docs would provide useful additional context.
